### DISCUSSION TITLE
how to add a header comment?

### DISCUSSION THREAD

**Discussion by tiansira** [numToStr/Comment.nvim#393] [Q&A]
does anyone know how to add a header comment?
like this. at the beginning of the file, one-click to add a header comment.
\###
 \# https://github.com/author: xxx
 \# @Date: xxx
 \# @LastEditTime: xxx
 \# @description:
\###

**Comment by gegoune** [comment]
Looks more like a task for snippet engine; or maybe something like Neogen?
